src/net.cpp

```diff
@@ -2830,18 +2830,14 @@ void CConnman::PushMessage(CNode* pnode, CSerializedNetMsg&& msg)
         pnode->mapSendBytesPerMsgCmd[msg.m_type] += nTotalSize;
         pnode->nSendSize += nTotalSize;
 
-        if (pnode->nSendSize > nSendBufferMaxSize)
-            pnode->fPauseSend = true;
+        if (pnode->nSendSize > nSendBufferMaxSize) pnode->fPauseSend = true;
         pnode->vSendMsg.push_back(std::move(serializedHeader));
-        if (nMessageSize)
-            pnode->vSendMsg.push_back(std::move(msg.data));
+        if (nMessageSize) pnode->vSendMsg.push_back(std::move(msg.data));
 
         // If write queue empty, attempt "optimistic write"
-        if (optimisticSend == true)
-            nBytesSent = SocketSendData(pnode);
+        if (optimisticSend == true) nBytesSent = SocketSendData(pnode);
```

Comment:
nit
```suggestion
        if (optimisticSend) nBytesSent = SocketSendData(pnode);
```

or:
```suggestion
        if (node->vSendMsg.empty()) nBytesSent = SocketSendData(pnode);
```
and remove the bool completely. The comment already clarifies what is happening.

Reply:
I'm more than happy to do this!